src/models/orderModel.ts

```diff
@@ -21,6 +23,13 @@ class OrderModel {
     return this.filled !== 0 ? this.remainingVolume / this.volume : 0;
   }
 
+  @computed
```

Comment:
how can i test it? ;-)

Reply:
then you don't need to add `accuracy` in the order modek, which is also wrong coz accuracy means nothing to the `Order`. The order only knows about the instrument